src/Anonymization/Anonymizer/Core/NullAnonymizer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\Core;
+
+use MakinaCorpus\DbToolsBundle\Anonymization\Anonymizer\AbstractAnonymizer;
+use MakinaCorpus\DbToolsBundle\Attribute\AsAnonymizer;
+use MakinaCorpus\QueryBuilder\Query\Update;
+
+#[AsAnonymizer(
+    name: 'null',
+    pack: 'core',
+    description: 'Set to NULL'
+)]
+class NullAnonymizer extends AbstractAnonymizer
+{
+    /**
+     * @inheritdoc
+     */
+    public function anonymize(Update $update): void
+    {
+        $expr = $update->expression();
+        $update->set(
+            $this->columnName,
+            $expr->raw('NULL'),
```

Comment:
Here you can write `$expr->null()` or simply a PHP `null` value, which will then be transparently propagated as an SQL `NULL` value.